.github/workflows/ci.yml

```diff
@@ -6,6 +6,10 @@ on:
   pull_request:
     branches: [ 'dblp' ]
 
+concurrency:
```

Comment:
What is this for?

Reply:
stop a previous build when pushing a new commit :)